app/middleware/error_handler.py

```
@@ -0,0 +1,513 @@
+    """
+    Standardized error response format.
+    
+    Ensures all error responses follow a consistent structure
+    with proper error codes, messages, and debugging information.
+    """
+    
+    @staticmethod
+    def create(
+        error_code: str,
+        message: str,
+        details: Optional[Dict[str, Any]] = None,
```

Ruff (0.11.9)

Use `dict` instead of `Dict` for type annotation

Replace with `dict`

(UP006)